invenio_communities/views/communities.py

```diff
@@ -138,5 +139,6 @@ def communities_settings_privileges(community=None, pid_value=None):
         # Pass permissions so we can disable partially UI components
         # e.g Settings tab
         permissions=community.has_permissions_to(['update']),
-        active_menu_tab="settings"
+        active_menu_tab="settings",
+        logo=logo.to_dict() if logo else None
```

Comment:
Interesting, so the logo is an object, can it be serialized by the view?

Reply:
The view is supposed to only read the object, that for we have [this function](https://github.com/inveniosoftware/invenio-communities/blob/master/invenio_communities/communities/services/service.py#L114-L127) in the service which is called int the `pass_community_logo` decorator and with the to_dict() we get the dict representation of the logo object to be used in the view. Hope this answers your question.